tuned/plugins/plugin_disk.py

```diff
@@ -361,6 +388,6 @@ def _get_scheduler_quantum(self, device, ignore_missing=False):
 		value = self._cmd.read_file(sys_file, no_error=ignore_missing).strip()
 		if len(value) == 0:
 			if not ignore_missing:
-				log.info("disk_scheduler_quantum option is not supported by this HW")
+				log.warn("disk_scheduler_quantum option is not supported by this HW")
```

Comment:
I think the info/warn should be consistent. E.g. in the `@command_get("spindown")` there is info level, but here in the `@command_get("scheduler_quantum")` is the warn level, why? Also are you using the info level for the 'get' and the warn level for the 'set'? Is there any reason for it? Or maybe the set/get levels should be also consistent?

Reply:
We were talking about "higher" level of log before, so changed it to warn, info in `@command_get("spindown")` was incaution, biut I changed it all to info now.